### PR TITLE
Add tutorial links to the docs #78073

### LINKED ISSUE #78073: Add tutorial links to the docs
### Summary

We have some great tutorials for community folks to use here:
https://www.ansible.com/products/ansible-community-training
We should add them to the docs :-)

Since most are focused on the developer experience, especially collections, we can add them to:
https://docs.ansible.com/ansible/devel/community/contributions_collections.html

### Issue Type

Documentation Report

### Component Name

docs/docsite/rst/community/contributions_collections.rst

### Ansible Version

```console
$ ansible --version
2.14
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
none
```

### OS / Environment

none

### Additional Information

none

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by paulreece**
##### SUMMARY
Added a small note to the Docs in order to link to tutorials.

`* See :ref:`Collection developer tutorials<https://www.ansible.com/products/ansible-community-training>_` for a quick introduction on how to develop and test your collection contributions.
`
Fixes #78073
##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
* docs/docsite/community/contributions_collections.rst

##### ADDITIONAL INFORMATION
Added a Doc fix

##### Before
```.. _collections_contributions:

*************************************
Ansible Collections Contributor Guide
*************************************

.. toctree::
   :maxdepth: 2

   collection_development_process
   reporting_collections
   create_pr_quick_start
   collection_contributors/test_index
   collection_contributors/collection_reviewing
   maintainers
   contributing_maintained_collections
   steering/steering_index
   documentation_contributions
   other_tools_and_programs



If you have a specific Ansible interest or expertise (for example, VMware, Linode, and so on, consider joining a :ref:`working group <working_group_list>`.

Working with the Ansible collection repositories
=================================================

* How can I find :ref:`editors, linters, and other tools <other_tools_and_programs>` that will support my Ansible development efforts?
* Where can I find  guidance on :ref:`coding in Ansible <developer_guide>`?
* How do I :ref:`create a collection <developing_modules_in_groups>`?
* How do I :ref:`rebase my PR <rebase_guide>`?
* How do I learn about Ansible's :ref:`testing (CI) process <developing_testing>`?
* How do I :ref:`deprecate a module <deprecating_modules>`?


```
##### After
```
.. _collections_contributions:

*************************************
Ansible Collections Contributor Guide
*************************************

.. toctree::
   :maxdepth: 2

   collection_development_process
   reporting_collections
   create_pr_quick_start
   collection_contributors/test_index
   collection_contributors/collection_reviewing
   maintainers
   contributing_maintained_collections
   steering/steering_index
   documentation_contributions
   other_tools_and_programs



If you have a specific Ansible interest or expertise (for example, VMware, Linode, and so on, consider joining a :ref:`working group <working_group_list>`.

Working with the Ansible collection repositories
=================================================

* How can I find :ref:`editors, linters, and other tools <other_tools_and_programs>` that will support my Ansible development efforts?
* Where can I find  guidance on :ref:`coding in Ansible <developer_guide>`?
* How do I :ref:`create a collection <developing_modules_in_groups>`?
* How do I :ref:`rebase my PR <rebase_guide>`?
* How do I learn about Ansible's :ref:`testing (CI) process <developing_testing>`?
* How do I :ref:`deprecate a module <deprecating_modules>`?
* See :ref:`Collection developer tutorials<https://www.ansible.com/products/ansible-community-training>_` for a quick introduction on how to develop and test your collection contributions.
```

### CODE REVIEW

File: docs/docsite/rst/community/contributions_collections.rst [hunk 1]
```diff
@@ -31,3 +31,4 @@ Working with the Ansible collection repositories
 * How do I :ref:`rebase my PR <rebase_guide>`?
 * How do I learn about Ansible's :ref:`testing (CI) process <developing_testing>`?
 * How do I :ref:`deprecate a module <deprecating_modules>`?
+* See :ref:`Collection developer tutorials<https://www.ansible.com/products/ansible-community-training>_` for a quick introduction on how to develop and test your collection contributions.
```
Comment (samccann) [inline docs/docsite/rst/community/contributions_collections.rst]:
```suggestion
* See :ref:`Collection developer tutorials<https://www.ansible.com/products/ansible-community-training>`_ for a quick introduction on how to develop and test your collection contributions.
```

File: docs/docsite/rst/community/contributions_collections.rst [hunk 2]
```diff
@@ -31,3 +31,4 @@ Working with the Ansible collection repositories
 * How do I :ref:`rebase my PR <rebase_guide>`?
 * How do I learn about Ansible's :ref:`testing (CI) process <developing_testing>`?
 * How do I :ref:`deprecate a module <deprecating_modules>`?
+* See :ref:`Collection developer tutorials<https://www.ansible.com/products/ansible-community-training>`_ for a quick introduction on how to develop and test your collection contributions.
```
Comment (samccann) [inline docs/docsite/rst/community/contributions_collections.rst]:
```suggestion
* See `Collection developer tutorials<https://www.ansible.com/products/ansible-community-training>`_ for a quick introduction on how to develop and test your collection contributions.
```

**Comment by samccann** [comment]
Thanks @paulreece ! I'm fixing the CI errors (I forgot  that :ref: is only for internal links). Thanks for taking this on!  When CI passes I can merge.

File: docs/docsite/rst/community/contributions_collections.rst [hunk 3]
```diff
@@ -31,3 +31,4 @@ Working with the Ansible collection repositories
 * How do I :ref:`rebase my PR <rebase_guide>`?
 * How do I learn about Ansible's :ref:`testing (CI) process <developing_testing>`?
 * How do I :ref:`deprecate a module <deprecating_modules>`?
+* See `Collection developer tutorials<https://www.ansible.com/products/ansible-community-training>`_ for a quick introduction on how to develop and test your collection contributions.
```
Comment (samccann) [inline docs/docsite/rst/community/contributions_collections.rst]:
```suggestion
* See `Collection developer tutorials <https://www.ansible.com/products/ansible-community-training>`_ for a quick introduction on how to develop and test your collection contributions.
```

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @paulreece for the Ansible docs fix and welcome to Ansible!

**Comment by paulreece** [comment]
Thanks @samccann glad to be here!

**Pull request merged**